### PR TITLE
Mob 405

### PR DESCRIPTION
**Pull request by NarendraPrabhu**
Changing Help & Support Activity title according to the client input extra param in intent -"lithium.community.android.sdk.ui.components.activities.LiSupportHomeActivity.PARAM_TITLE", otherwise fallback value will be `Camelcase(${tenant-id})+" Community"`.

### CODE REVIEW

File: li-android-sdk-ui/build.gradle [hunk 1]
```diff
@@ -96,16 +96,13 @@ android {
...
-    compile('lithium.community.android.sdk:li-android-sdk-core:1.3.0+c3c7aa3', {
-        changing = true
-    })
-    compile 'com.android.support:cardview-v7:27.0.2'
-    compile 'com.android.support:design:27.0.2'
-    compile 'com.android.support:recyclerview-v7:27.0.2'
+    compile "com.android.support:design:27.0.2"
     compile 'com.google.code.gson:gson:2.8.0'
     compile 'com.squareup.picasso:picasso:2.5.2'
+    compile('lithium.community.android.sdk:li-android-sdk-core:1.3.0+', {
```
Comment (adityasharat) [inline li-android-sdk-ui/build.gradle]: Is it a good idea to use `+` in the version? This is generally not recommended

File: li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java [hunk 2]
```diff
@@ -45,6 +47,9 @@
  * and {@link LiCreateMessageFragment} as two different tabs
  */
 public class LiSupportHomeActivity extends AppCompatActivity {
```
Comment (adityasharat) [inline li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java]: There seem to be a lot of code style issues (primarily spacing and indentation). Use `./gradlew checkstyle` to identify them.

File: li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java [hunk 3]
```diff
@@ -110,9 +115,19 @@ protected void onCreate(Bundle savedInstanceState) {
...
         Intent intent = getIntent();
         Bundle bundle = intent.getExtras();
+        String title = null;
         if (bundle != null) {
             ssoToken = bundle.getString(LiCoreSDKConstants.LI_SSO_TOKEN, null);
+            title = bundle.getString(EXTRA_PARAM_TITLE);
+        }
+
+        if(TextUtils.isEmpty(title)) {
+            String titleParam = convertCamelcase(LiSDKManager.getInstance().getCredentials().getTenantId());
```
Comment (adityasharat) [inline li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java]: I suggest we use `LiSDKManager.getInstance().getCredentials().getClientName()` as this is a developer set user friendly name!

File: li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java [hunk 4]
```diff
@@ -153,6 +168,21 @@ protected void onDestroy() {
         unregisterReceiver(createMessageSuccessReceiver);
     }
 
+    static private String convertCamelcase(String value){
```
Comment (adityasharat) [inline li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java]: This won't be required if we use `LiSDKManager.getInstance().getCredentials().getClientName()`

File: li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java [hunk 5]
```diff
@@ -110,9 +115,19 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        String title = null;
         if (bundle != null) {
             ssoToken = bundle.getString(LiCoreSDKConstants.LI_SSO_TOKEN, null);
+            title = bundle.getString(EXTRA_PARAM_TITLE);
+        }
+
+        if(TextUtils.isEmpty(title)) {
+            String titleParam = convertCamelcase(LiSDKManager.getInstance().getCredentials().getTenantId());
+            if (!TextUtils.isEmpty(titleParam)) {
+                title = String.format(getString(R.string.li_support_activity_custom_title), titleParam);
```
Comment (adityasharat) [inline li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java]: Might as well use [Android parameterized string resources](https://developer.android.com/guide/topics/resources/string-resource)

File: li-android-sdk-ui/build.gradle (hunk 1 above)
Comment (NarendraPrabhu) [reply li-android-sdk-ui/build.gradle]: this version issue is tracked in a different issue

File: li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java (hunk 2 above)
Comment (NarendraPrabhu) [reply li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java]: sure

File: li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java (hunk 3 above)
Comment (NarendraPrabhu) [reply li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java]: As I explained client-name or api-app-name is not necessarily the tenant name, amongst the different parameters tenant-id is the most nearest to the community name.

File: li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java (hunk 5 above)
Comment (NarendraPrabhu) [reply li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java]: yes, sure

File: li-android-sdk-ui/build.gradle (hunk 1 above)
Comment (adityasharat) [reply li-android-sdk-ui/build.gradle]: Understood!

File: li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java (hunk 2 above)
Comment (NarendraPrabhu) [reply li-android-sdk-ui/src/main/java/lithium/community/android/sdk/ui/components/activities/LiSupportHomeActivity.java]: resolved

**Pull request merged**